contracts/rootchain/RootChain.sol

```diff
@@ -262,6 +262,48 @@ contract RootChain is Ownable {
         return true;
     }
 
+    // Validator of any block can call this function to exit the fees collected
+    // for that particular block. The validator declares the fee amount, and the exit
+    // is added to exit queue with the lowest priority for that block.
+    // In case of a wrong fee amount specified, users on the Plasma chain would mass exit.
+    // In case of the fee UTXO already spent, anyone can challenge the fee exit by providing
+    // the spend of the fee UTXO.
+    // @param blockNumber      the block for which the validator wants to exit fees
+    // @param feeAmount        the amount of the fee collected in block blockNumber
+    function startFeeExit(uint256 blockNumber, uint256 feeAmount)
+        public
+        payable
+        onlyOwner
+    {
+        // specified blockNumber must exist in child chain
+        bytes32 empty;
+        require(childChain[blockNumber].root != empty, "specified block does not exist in child chain.");
+
+        require(msg.value >= minExitBond, "insufficient exit bond");
+        if (msg.value > minExitBond) {
+            uint256 excess = msg.value.sub(minExitBond);
+            balances[msg.sender] = balances[msg.sender].add(excess);
+            totalWithdrawBalance = totalWithdrawBalance.add(excess);
+        }
+
+        // a fee UTXO has explicitly defined position [blockNumber, 2**16 - 1, 0]
+        uint256 txIndex = 2**16 - 1;
```

Comment:
We are doing this, can we need to enforce a this maximum block size in `submitBlock`

Reply:
also please add a test to make sure it reverts if block size is incorrect

Reply:
@laskdaf what do you mean? block size is not passed into `startFeeExit`, only the block number. I don't think there's a way for us to check a fee exit's corresponding block size against the true block size.

Reply:
@hamdiallam from the discord discussion, I understand that we're not going to enforce maximum block size in `submitBlock`?

Reply:
@AdityaSripal numTxns[i] less or equal to 2 ** 16 - 1, right? because the fee UTXO is transaction number 2 ** 16 (and has index 2 ** 16 - 1).